.pipelines/templates/patch-images.yaml

```diff
@@ -0,0 +1,61 @@
+pr: none
```

Comment:
@ashnamehrotra this is an ADO pipeline, GHA won't work here since copa is not an ADO task